java-frontend/src/test/java/org/sonar/java/se/ProgramStateDataProvider.java

```diff
@@ -64,8 +64,7 @@ public String stack() {
   }
 
   public String lastEvaluatedSymbol() {
```

Comment:
Rename this method, even if it's only used for the viewer. let's keep it coherent.